proto/cacheclient.proto

```diff
@@ -435,6 +437,47 @@ message _ListLengthResponse {
   }
 }
 
+message _ListTrimRequest {
+  bytes list_name = 1;
+  optional sint32 start = 2;
+  optional sint32 end = 3;
+  optional sint32 step = 4;
+}
+
+message _ListTrimResponse {
+  message _Found {
+    repeated bytes values = 1;
+  }
+
+  message _Missing {}
+
+  oneof list {
+    _Found found = 1;
+    _Missing missing = 2;
+  }
```

Comment:
I believe this should just a be a simple empty response? Receipt of the response would indicate a successful trim, otherwise it would return an error?